internal/server/models/deviceflowauthrequest.go

```diff
@@ -24,4 +24,7 @@ type DeviceFlowAuthRequest struct {
 
 	// AccessKey will be populated by some queries, but is never used on writes.
 	AccessKey *AccessKey `db:"-"`
+
+	// Organization will be populated by some queries, but is never used on writes.
+	Organization *Organization `gorm:"-" db:"-"`
```

Comment:
I had to add the `gorm` tag here as well, without it I was getting errors. I think because the `data.GetDeviceFlowAuthRequest` function uses gorm for selects, so we have to exclude the field.  I'm not sure why the `AccessKey` field above does not have the same problem.

Reply:
Shouldn;t need this anymore, I'll remove it